dcos/package.py

```diff
@@ -943,6 +944,25 @@ def copy_to_cache(self, target_dir):
                 'Unable to fetch packages from [{}]'.format(self.url))
 
 
+def _onerror(func, path, exc_info):
```

Comment:
Why is the ~/.git directory created read-only?  If our user clones the repo on windows, shouldn't the user have write access?